crates/typst-library/src/math/style.rs

```diff
@@ -38,44 +38,42 @@ pub fn italic(
     body.styled(EquationElem::set_italic(Smart::Custom(true)))
 }
 
-/// Serif (roman) font style in math.
+/// 数式中のセリフ（ローマン）フォントスタイル。
 ///
-/// This is already the default.
+/// これがデフォルトです。
 #[func(keywords = ["mathrm"])]
 pub fn serif(
-    /// The content to style.
+    /// スタイルを適用するコンテンツ。
     body: Content,
 ) -> Content {
     body.styled(EquationElem::set_variant(MathVariant::Serif))
 }
 
-/// Sans-serif font style in math.
+/// 数式中のサンセリフフォントスタイル。
 ///
 /// ```example
 /// $ sans(A B C) $
 /// ```
 #[func(title = "Sans Serif", keywords = ["mathsf"])]
 pub fn sans(
-    /// The content to style.
+    /// スタイルを適用するコンテンツ。
     body: Content,
 ) -> Content {
     body.styled(EquationElem::set_variant(MathVariant::Sans))
 }
 
-/// Calligraphic font style in math.
+/// 数式中のカリグラフィーフォントスタイル。
 ///
 /// ```example
 /// Let $cal(P)$ be the set of ...
 /// ```
 ///
-/// This corresponds both to LaTeX's `\mathcal` and `\mathscr` as both of these
-/// styles share the same Unicode codepoints. Switching between the styles is
-/// thus only possible if supported by the font via
-/// [font features]($text.features).
+/// このスタイルはLaTeXの`\mathcal`と`\mathscr`の両方に対応します。
+/// これは両スタイルが同じUnicodeのコードポイントを共有しているためです。
+/// このため、スタイル間の切り替えは[フォントフィーチャー]($text.features)を用いてサポートされているフォントでのみ可能です。
 ///
-/// For the default math font, the roundhand style is available through the
-/// `ss01` feature. Therefore, you could define your own version of `\mathscr`
-/// like this:
+/// デフォルトの数式フォントでは、ラウンドハンドスタイルが`ss01`フィーチャーとして利用可能です。
```

Comment:
英語としてroundhandは一般的なようんですが、その訳として「ラウンドハンド」や「ラウンドハンドスタイル」は一般的ではないようです。補足を付けてみてはいかがでしょうか？

```suggestion
/// デフォルトの数式フォントでは、ラウンドハンドスタイル（丸みを帯びた筆記体）が`ss01`フィーチャーとして利用可能です。
```